### ISSUE TITLE
Empty room displayed with no way to join or remove it

### ISSUE THREAD

**Issue by JaCoB1123** [vector-im/element-web#19893]
### Steps to reproduce

1. When I open Element I see an empty room with a red exclamation mark next to it. 
2. When I select it, it allows me to either send a bug report or delete my cache and reload Element. 

I did clean the cache multiple times. The problem also persists after logging in again or using Element Web.

#### What did you expect?
I expect to be either able to join the room or having it cleaned up, if it no longer exists.

#### What happened instead?
Empty room is always displayed, with no way to interact with it.

### Operating system

Windows/Firefox

### Application version

Version von Element: 1.9.3 Version von Olm: 3.2.3

### How did you install the app?

scoop

### Homeserver

matrix.javil.eu

### Will you send logs?

Yes

**Comment by gsouquet** [comment]
@JaCoB1123 your bug report says that you will send logs but I don't think you have already.
It would be very useful to have them to understand what is going wrong here

**Comment by JaCoB1123** [comment]
@gsouquet I did send logs directly after submitting the issue.
The problem seems to have solved itself for me today, by setting up my new phone. I had two room invitations there, which I was able to join. The empty rooms have now disappeared on my desktop.
I just sent the logs again, I don't know if the issue is still contained there though. Feel free to close, if it's not visible in the logs - I'll reopen if necessary.
